internal/tenants/tokens.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2022 Namespace Labs Inc; All rights reserved.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+
+package tenants
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+
+	"namespacelabs.dev/foundation/internal/auth"
+	"namespacelabs.dev/foundation/internal/fnapi"
+	"namespacelabs.dev/foundation/internal/fnerrors"
+	"namespacelabs.dev/foundation/internal/github"
+	"namespacelabs.dev/foundation/internal/workspace/dirs"
+)
+
+const (
+	tokenTxt          = "tenant_token.txt"
```

Comment:
Please use json--makes structure clearer and is consistent with our other configuration files.